src/interfaces/IOwnerManager.sol

```diff
@@ -0,0 +1,7 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.5.0;
+
+interface IOwnerManager {
```

Comment:
Is this needed? Can't it just be part of the ISafe interface?

Reply:
Yes they were copied from the safe.